docs/conf.py

```diff
@@ -151,12 +170,33 @@
 #  dir menu entry, description, category)
 texinfo_documents = [
     (master_doc, 'minos',
-     'Minos Microservice SAGA Documentation',
+     'Minos Microservice Common Documentation',
```

Comment:
Shouldn't this be `Saga`?
```suggestion
     'Minos Microservice Saga Documentation',
```